src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml

```diff
@@ -93,4 +93,5 @@ AutoRun:
           - shard-lite-80-feature-flags
       branch_name:
         $neq:
-          - v4.0
+          - v4.4
```

Comment:
Do we run this on the 4.4 branch? Why?

Reply:
@afuschetto we do still have [4.4 branch for sys-perf project in Evergreen CI](https://spruce.mongodb.com/commits/sys-perf-4.4). The last executed patch on this branch was on May 7th, 2024.

Not sure why we still run 4.4 system performance patches